modules/sandboxed-containers-enable-debug-crio.adoc

```diff
@@ -6,68 +6,36 @@
 [id="sandboxed-containers-enable-debug-logs_{context}"]
 = Enabling debug logs for {sandboxed-containers-first}
 
-As a cluster administrator, you can collect a more detailed level of logs for {sandboxed-containers-first}. Enhance logging by changing the `log_level` in the CRI-O runtime for the worker nodes running {sandboxed-containers-first}.
+As a cluster administrator, you can collect a more detailed level of logs for {sandboxed-containers-first}. Enhance logging by changing the `logLevel` field in the KataConfig CR. This changes the `log_level` in the CRI-O runtime for the worker nodes running {sandboxed-containers-first}.
 
 .Procedure
 
-. Create a YAML file for the `ContainerRuntimeConfig` CR with the following manifest:
-+
-[source,yaml]
-----
-apiVersion: machineconfiguration.openshift.io/v1
-kind: ContainerRuntimeConfig
-metadata:
- name: crio-debug
-spec:
- machineConfigPoolSelector:
-  matchLabels:
-    pools.operator.machineconfiguration.openshift.io/worker: '' <1>
- containerRuntimeConfig:
-    logLevel: debug
-----
-<1> Specify a label for the machine config pool that you want you want to modify.
+. Change the `logLevel` field in your existing KataConfig CR to `debug`:
 
-. Create the `ContainerRuntimeConfig` CR:
-+
 [source,terminal]
 ----
-$ oc create -f ctrcfg.yaml
+$ oc patch kataconfig cluster-kataconfig --type merge --patch '{"spec":{"logLevel":"debug"}}'
 ----
 
-+
 [NOTE]
 ====
-The file name listed above is a suggestion. You can create this file using another name.
+The object name for the KataConfig CR (`cluster-kataconfig`) listed above is the suggested name when creating your KataConfig CR. You can create this object using another name, however, make sure you reference the correct object name when running the above command.
```

Comment:
The SSG states to "Avoid directional indicators such as left, right, above and below when providing navigational instruction in the user interface." See, https://redhat-documentation.github.io/supplementary-style-guide/#cloud-services-visual-info

Reply:
This " however, make sure you reference the correct object name when running the above command." confused me a bit. I wonder if you could say something like 

> You can create this object using another name as long as you reference the correct object name when running the command to create the `ContainerRuntimeConfig` CR.

Or something like that.

Reply:
This is a great suggestion, thanks!